.github/workflows/Basic.yml

```diff
@@ -11,6 +11,12 @@ jobs:
     steps:
       - name: Checkout sources
         uses: actions/checkout@v2
+      - name: Fix git safe.directory in container
```

Comment:
What's this for?

Reply:
This is because the submodules checkout action (srt32/git-actions@v0.0.3) complains about an unsafe workspace directory when checking out submodules.

See by example https://github.com/InjectiveLabs/cw-injective/actions/runs/4346577315/jobs/7592812443.

That ugly command is creating the git directive to accept an unsafe workspace dir, for the docker container workspace dir.